src/EffectComposer.tsx

```diff
@@ -119,13 +120,13 @@ export const EffectComposer = React.memo(
 
       const group = useRef(null)
       useLayoutEffect(() => {
-        let effectPass
-        if (group.current && group.current.__r3f && composer) {
+        let effectPass: EffectPass
+        if (group.current && (group.current as any).__r3f && composer) {
           effectPass = new EffectPass(camera, ...(group.current as any).__r3f.objects)
           effectPass.renderToScreen = true
           composer.addPass(effectPass)
```

Comment:
I'd prefer `useInstanceHandle` if you must access internal fields. In v9, this is much more useful and stable for this case.

https://github.com/pmndrs/react-three-fiber/pull/2484

```jsx
const ref = React.useRef()
const instance = useInstanceHandle(ref)

React.useLayoutEffect(() => {
  instance.current.parent.foo()
}, [])

<group ref={ref} />
```